torch/__init__.py

```diff
@@ -138,18 +138,15 @@ def set_default_tensor_type(t):
 
         >>> torch.set_default_tensor_type("torch.FloatTensor")
         >>> torch.Tensor([1.2, 3])
-
-         1.2000
-         3.0000
-        [torch.FloatTensor of size (2,)]
+        tensor([ 1.2000, 3.0000])
 
         >>> torch.set_default_tensor_type(torch.double)
         >>> torch.Tensor([1.2, 3])
+        tensor([ 1.2000, 3.0000], dtype=torch.float64)
 
-         1.2000
-         3.0000
-        [torch.DoubleTensor of size (2,)]
-
+        >>> torch.set_default_tensor_type(torch.cuda.double)
+        >>> torch.Tensor([2, 3])
+        tensor([ 2, 3], dtype=torch.cuda.float64)
```

Comment:
Aren't we separating device from dtype? Might be better to modify the print accordingly?

Reply:
The device will be separated from dtype after #6470 . This PR needs to be updated when that merges...

Reply:
Yeah, should be a small update because CPU will be the implicit, so only this example will need to be updated.